Gemfile

```diff
@@ -55,10 +55,11 @@ end
 #
 
 group :benchmark do
+  gem "ruby-prof"
+
   if ENV["BENCHMARK"]
     gem "benchmark-ips"
     gem "rbtrace"
-    gem "ruby-prof"
```

Comment:
What's this about?

Reply:
so that one doesn't have to run
```
BENCHMARK=1 script/rubyprof
```
or
```
BENCHMARK=1 bundle exec ruby script/rubyprof
```

Reply:
So, should we just get rid of the `if ENV["BENCHMARK"]` guard altogether?

Reply:
no I prefer having that guard because, `stackprof` cannot be installed on Windows.. and this guard ensures it's not looked for by Bundler

Reply:
So change the guard to look for Windows?

Reply:
That would be a nice alternative, but I do not want to change all that in this PR.
This PR is dealing with just `script/rubyprof` that seemed to have been left unused.., so, changes to it wouldn't disrupt an existing workflow..

Reply:
Let’s revert this and discuss it in a separate PR then?

Reply:
Ping: @parkr, @oe, @DirtyF  Would like your opinion on this..
Which of the following formats do you prefer?
```
$ BENCHMARK=1 script/rubyprof
```
```
$ script/rubyprof
```
*I'll revert the change to the Gemfile only if the majority prefers the former format*

Reply:
I’m not trying to start a war. I’m just trying to understand the motivation for the change.

Reply:
okay.. I'll concede and revert this change for a future discussion..